### ISSUE TITLE
Can't build / install zipfile using node 6.8.1 / npm 3.10.8

### ISSUE THREAD

**Issue by ax487** [mapbox/node-zipfile#70]
I tried to install `zipfile` globally using npm:

```
npm install zipfile --global
```

Unfortunately I get some build errors. As far as I can see the compilation of the C++ part of the library fails:

```
../node_modules/nan/nan.h:602:20: error: variable or field ‘AddGCEpilogueCallback’ declared void
       v8::Isolate::GCEpilogueCallback callback
                    ^~~~~~~~~~~~~~~~~~
../node_modules/nan/nan.h:602:7: error: ‘GCEpilogueCallback’ is not a member of ‘v8::Isolate’
       v8::Isolate::GCEpilogueCallback callback
       ^~
../node_modules/nan/nan.h:603:18: error: expected primary-expression before ‘gc_type_filter’
     , v8::GCType gc_type_filter = v8::kGCTypeAll) {
                  ^~~~~~~~~~~~~~
../node_modules/nan/nan.h:608:20: error: variable or field ‘RemoveGCEpilogueCallback’ declared void
       v8::Isolate::GCEpilogueCallback callback) {
                    ^~~~~~~~~~~~~~~~~~
../node_modules/nan/nan.h:608:7: error: ‘GCEpilogueCallback’ is not a member of ‘v8::Isolate’
       v8::Isolate::GCEpilogueCallback callback) {
       ^~
../node_modules/nan/nan.h:613:20: error: variable or field ‘AddGCPrologueCallback’ declared void
       v8::Isolate::GCPrologueCallback callback
```

Relevant part of the `npm-debug.log`:

```
7894 verbose stack Error: zipfile@0.5.9 install: `node-pre-gyp install --fallback-to-build`
7894 verbose stack Exit status 1
7894 verbose stack     at EventEmitter.<anonymous> (/usr/lib/node_modules/npm/lib/utils/lifecycle.js:255:16)
7894 verbose stack     at emitTwo (events.js:106:13)
7894 verbose stack     at EventEmitter.emit (events.js:191:7)
7894 verbose stack     at ChildProcess.<anonymous> (/usr/lib/node_modules/npm/lib/utils/spawn.js:40:14)
7894 verbose stack     at emitTwo (events.js:106:13)
7894 verbose stack     at ChildProcess.emit (events.js:191:7)
7894 verbose stack     at maybeClose (internal/child_process.js:877:16)
7894 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:226:5)
7895 verbose pkgid zipfile@0.5.9
7896 verbose cwd /home/username/Downloads/tileoven
7897 error Linux 4.7.6-1-ARCH
7898 error argv "/usr/bin/node" "/usr/bin/npm" "install" "-g" "zipfile"
7899 error node v6.8.1
7900 error npm  v3.10.8
7901 error code ELIFECYCLE
7902 error zipfile@0.5.9 install: `node-pre-gyp install --fallback-to-build`
7902 error Exit status 1
7903 error Failed at the zipfile@0.5.9 install script 'node-pre-gyp install --fallback-to-build'.
7903 error Make sure you have the latest version of node.js and npm installed.
7903 error If you do, this is most likely a problem with the zipfile package,
7903 error not with npm itself.
7903 error Tell the author that this fails on your system:
7903 error     node-pre-gyp install --fallback-to-build
7903 error You can get information on how to open an issue for this project with:
7903 error     npm bugs zipfile
7903 error Or if that isn't available, you can get their info via:
7903 error     npm owner ls zipfile
7903 error There is likely additional logging output above.
7904 verbose exit [ 1, true ]
```

**Comment by jazzzz** [comment]
It should work on zipfile 0.5.10 which in not released on npm yet (see # #71).

**Comment by springmeyer** [comment]
v0.5.10 is now published and should be working.
